proposal.md

```diff
@@ -1 +1,100 @@
-**Include your feature proposal here**
+This proposal is a mostly a list of ideas thrown together, there is much to discuss. Basically, the goal is to add support for multiple workspaces, add flexibility on authentication and make it easier to implement at scale.
+
+## Terms
+
+- Chat Users: refers to users speaking with a bot (which may or may not be authenticated)
+- Collaborators: Authenticated users on the studio
+
+## Summary of proposed changes
+
+- Adds support for multiple workspaces (a workspace has custom user roles and hosts multiple bots)
+- Remove collaborators from "workspaces.json" and move them to the database
+- Collaborators accounts are global (may be in one or more workspaces)
+- Extract "Auth Strategies" from botpress config into its own file (there could be multiple basic auth or saml auth config)
```

Comment:
I don't see the point of doing that, we want to keep things in `botpress.config.json`.  Furthermore we want to change the content of `botpress.config.json` depending on the env (e.g. `dev.botpress.config.json`, `production.botpress.config.json`) which would not work if we split it into different files

Reply:
We could definitely add it in the botpress config, eg transform the authStrategy from a single object to an array of strategies, Just thought there was already a lot of stuff in the config file :)

Reply:
it's getting big but we'll have a GUI :-)